_posts/2023-06-16-deploy-on-kubernetes-with-helm.adoc

```diff
@@ -0,0 +1,400 @@
+
+We will use https://kind.sigs.k8s.io/[kind] as our Kubernetes cluster.
+
+## Install Helm Repository for WildFly Chart
+
+The `wildfly` Chart can be installed from the https://docs.wildfly.org/wildfly-charts/ repository
+
+[source,nowrap]
+----
+$ helm repo add wildfly https://docs.wildfly.org/wildfly-charts/
+"wildfly" has been added to your repositories
+$ helm search repo wildfly
+NAME                  	CHART VERSION	APP VERSION	DESCRIPTION                                       
+wildfly/wildfly       	2.3.1        	           	Build and Deploy WildFly applications on OpenShift
+wildfly/wildfly-common	2.1.0        	           	A library chart for WildFly-based applications  
+----
+
+## Make sure your application pom contains the wildfly-maven-plugin plugin
+
+In our helloworld quickstart the wildfly-maven-plugin plugin is already present under the 'openshift' profile.
+Add it if you don't have it in your pom.xml.
+
+[source,nowrap]
+----
+<profiles>
+...
+        <profile>
+            <id>openshift</id>
+            <build>
+                <plugins>
+                    <plugin>
+                        <groupId>org.wildfly.plugins</groupId>
+                        <artifactId>wildfly-maven-plugin</artifactId>
+                        <version>${version.wildfly.maven.plugin}</version>
+                        <configuration>
+                            <feature-packs>
+                                <feature-pack>
+                                    <location>org.wildfly:wildfly-galleon-pack:${version.server}</location>
+                                </feature-pack>
+                                <feature-pack>
+                                    <location>org.wildfly.cloud:wildfly-cloud-galleon-pack:${version.cloud.fp}</location>
+                                </feature-pack>
+                            </feature-packs>
+                            <layers>
+                                <layer>cloud-server</layer>
+                            </layers>
+                            <filename>${project.artifactId}-${project.version}.war</filename>
+                        </configuration>
+                        <executions>
+                            <execution>
+                                <goals>
+                                    <goal>package</goal>
+                                </goals>
+                            </execution>
+                        </executions>
+                    </plugin>
+                </plugins>
+            </build>
+        </profile>
+...
+    </profiles>    
+----
+
+Note: In our helloworld quickstart the maven-war-plugin is used to create the war. If you want to use a different plugin to create the war or jar you need to set the 'filename' configuration of the wildfly-maven-plugin plugin accordingly.
+
+## Build the docker image with the wildfly-maven-plugin plugin 
+
+Note: In this example we use 'docker', but 'podman' works as well.
```

Comment:
How does this note apply to the subsequent block? I see in the log output that the wildfly-maven-plugin is using docker, but how does the user use podman?

I'm pretty sure the answer is podman aliases the docker commands so it just works, in which case perhaps say:

Note: In this example the wildfly-maven-plugin calls 'docker', but if you have 'podman' installed that will work as well.